### PR TITLE
Add helm charts. Add support for projected service account tokens 

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
Add helm charts. Drop kustomization `config` folder.
Add support for projected service account tokens for the `manager`, as well as for the `kube-rbac-proxy` container of the `terminal-controller-manager` deployment.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```breaking operator
The kustomization `config` folder was dropped. Instead, a helm chart was added `charts/terminal` to install the `terminal-controller-manager`.
```

```feature operator
Added support for projected service account tokens for the `manager`, as well as for the `kube-rbac-proxy` container of the `terminal-controller-manager` deployment.
```

### CODE REVIEW

**Review by grolu — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -6,116 +6,88 @@
...
 Host and target cluster can also be the same. For more details and a usage scenario [see docs here](https://github.com/gardener/dashboard/blob/master/docs/Webterminals.md).
 
 ## Development Setup
 
 Prerequisites:
-- [golang](https://golang.org/dl/) 
+- [golang](https://golang.org/dl/)
 - [kind](https://github.com/kubernetes-sigs/kind)
-- [kustomize](https://github.com/kubernetes-sigs/kustomize)
+- [helm](https://helm.sh/docs/intro/install/)
```
Comment (holgerkoser) [inline README.md:19]: May be add that `go env GOPATH` must be in the `PATH`

**Review by holgerkoser — APPROVED** [review]
\lgtm

File: README.md (hunk 1 above)
Comment (petersutter) [reply README.md:19]: I have fixed the `hack/golangci-lint.sh` script to take the binary from the GOPATH https://github.com/gardener/terminal-controller-manager/pull/134/commits/57cf22f42608b01cf3e65fa04c6c84518645ae01
so either running this script or `make lint` should be sufficient

**Pull request merged**